config/default/kustomization.yaml

```diff
@@ -1,16 +1,15 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-# Adds namespace to all resources. Keep it in capm3-system, as it is a
-# dependency for CAPM3
-namespace: capm3-system
+# Adds namespace to all resources.
+namespace: ipam-system
```

Comment:
Why `ipam-system`? I think it is a bit risky to make it so generic. There could be other IPAMs that would use the same name. Maybe `metal3-ipam-system`?